pkgs/development/libraries/mesa/default.nix

```diff
@@ -272,6 +282,8 @@ self = stdenv.mkDerivation {
     (lib.mesonOption "video-codecs" "all")
   ] ++ lib.optionals (vulkanLayers != []) [
     (lib.mesonOption "vulkan-layers" (builtins.concatStringsSep "," vulkanLayers))
+  ] ++ lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
```

Comment:
Should this be `canExecute`?

Reply:
Not sure, it might get messy with how I'm pulling intel_clc from a native build.

Reply:
I don't see how? You'd just not pull it in more cases.

Reply:
Tested the usual configs and it does.